src/app/containers/RadioSchedule/index.jsx

```diff
@@ -19,8 +24,14 @@ const RadioScheduleContainer = ({ endpoint }) => {
         .then(handleResponse)
         .catch(e => logger.error(`HTTP Error: "${e}"`));
 
-    fetchRadioScheduleData(endpoint);
-  }, [endpoint]);
+    if (enabled) {
```

Comment:
Should this take `hasRadioSchedule` into account as well? If so it could probably be placed in a variable above `const radioScheduleEnabled = enabled && hasRadioSchedule`

And place this on line 32
```js
if (!radioScheduleEnabled) {
  return null;
}
```

Reply:
Made those changes